carbon-design-system/carbon-addons-iot-react
# Repo: upgrade Carbon to 10.53

Closes #

**Summary**

#### 10.50
- changed publicAPI.test to update warning message from MultiSelect.Filterable

#### 10.51
- no test failures -- updated snapshots

#### 10.52
- Had to add mocks for window.mediaMatches, but otherwise no new test failures

#### 10.53
- No new test failures

**Change List (commits, features, bugs, etc)**

- upgraded to carbon 10.53
- added mock for window.matchMedia
- updated copy for warning message for deprecated MultiSelect.Filterable
- replaced temp ref div with forwardedRef in Table/Pagination
- replaced useDropDownTitleFixer hook with `renderSelectedItem` in new version of Carbon

**Acceptance Test (how to verify the PR)**

- check titles on selected elements in ColorDropdown, IconDropdown, SimpleIconDropdown, TableViewDropdown
- check table pagination resizes correctly when below 500px (still have open issue to adjust this minimum)

**Regression Test (how to make sure this PR doesn't break old functionality)**

- standard carbon upgrade work to check stories



**Things to look for during review**
